grobid-core/src/main/java/org/grobid/core/jni/JEPThreadPool.java

```diff
@@ -64,40 +111,20 @@ private JEPThreadPool() {
      * (or create one the first time). 
      */
     public Jep getJEPInstance() {
-        if (jepInstances.get(Thread.currentThread().getId()) == null) {
-            JepConfig config = new JepConfig();
-            try {
-                File delftPath = new File(GrobidProperties.getInstance().getDeLFTFilePath());
-                if (!delftPath.exists()) {
-                    throw new GrobidResourceException("DeLFT installation path does not exist");
-                }
-                if (!delftPath.isDirectory()) {
-                    throw new GrobidResourceException("DeLFT installation path is not a directory");
-                }
-                config.addIncludePaths(delftPath.getAbsolutePath());
-                config.setClassLoader(Thread.currentThread().getContextClassLoader());
-                //System.out.println("jep instance thread: " + Thread.currentThread().getId());
-                try(Jep jep = new Jep(config)) {
```

Comment:
This was closing the `jep` instance after the initialisation.

Reply:
yes, I have not reviewed that branch